frontend/src/HomePage/AppCard.jsx

```diff
@@ -52,7 +52,8 @@ export default function AppCard({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isHovered]);
 
-  const updated = moment(app.created_at).fromNow(true);
+  const updated_at = app?.editing_version?.updated_at || app?.updated_at;
```

Comment:
Question: are we showing last updated user?
Shows in UI that
updated just now by `A`.
where `A` is app created user, and recently edited by other user

Reply:
@thehacklady